support/docker/production/.env

```diff
@@ -20,6 +20,9 @@ PEERTUBE_SMTP_TLS=false
 PEERTUBE_SMTP_DISABLE_STARTTLS=false
 PEERTUBE_ADMIN_EMAIL=<MY EMAIL ADDRESS>
 POSTFIX_myhostname=<MY DOMAIN>
+# If you need to generate a list of sub/DOMAIN keys
+# pass them as a whitespace separated string <DOMAIN>=<selector>
+OPENDKIM_DOMAINS=<MY DOMAIN>=peertube
```

Comment:
I think we should disable DKIM by default

Reply:
Without DKIM signature, as Postfix is the default smtp relay, DMARK verification fails so every email go into the spams, that's my initial though about enable DKIM. There has been 2 fixes on the image `mwader/postfix-relay`.

Reply:
I'm not sure to understand: most people won't have a DMARC record. While I agree this is better to have SPF/DKIM and DMARC, we cannot force it by default.

Reply:
Enabling DKIM with that image simply means that DKIM signature will be sent according to the RSA key automatically generated. 

- With DKIM enabled and no DMARC record, RSA keys will be generated, mails sent but no guarantee to not be received as a spam.
- Without DKIM enabled and no DMARC record, no RSA keys, mails are sent and the same thing happens : spam.

Enable it won't change the final behaviour, sending mail even if there is no DMARC record.

I think enabling that feature is not forcing to set a DMARC record but showing a way for a good practice with Postfix as a smtp.

I also think that if Postfix is just partially setup (as a local smtp), we also should disable it by default and let in any case the user configure all the mail part alone.